README.md

```diff
@@ -17,6 +17,29 @@ It requires python 3.6 or above, as well as the modules:
 In order to process ``.acq`` files, it needs [`bioread`](https://github.com/uwmadison-chm/bioread), an excellent module
 that can be found at [this link](https://github.com/uwmadison-chm/bioread).
 
+Linux and mac installation:
+Donwload the package as zip from github and uncompress or if you have ``git`` use the command:
+
+`` git clone https://github.com/smoia/phys2bids.git``
+
+open a terminal in the phy2bids folder and execute the command:
+
+``sudo python3 setup.py install``
+
+type the command:
+
+``phys2bids``
+
+if your output is:
+
+``usage: phys2bids [-h] -in FILENAME [-info] [-indir INDIR] [-outdir OUTDIR]
+                 [-heur HEUR_FILE] [-sub SUB] [-ses SES] [-chtrig CHTRIG]
+                 [-chsel [CHSEL [CHSEL ...]]] [-ntp NUM_TPS_EXPECTED] [-tr TR]
+                 [-thr THR] [-tbhd [TABLE_HEADER [TABLE_HEADER ...]]] [-v]
+phys2bids: error: the following arguments are required: -in/--input-file``
```

Comment:
The output should be the current version. Can you try it and modify accordingly please?